src/bot/scripts/card.js

```diff
@@ -728,946 +66,1233 @@ const card = new Command({
   preload: true,
 
   // -card <name>
-  cb: async function (client, message, guild) {
-
-    // deep copy cards so we can replace shit
-    // without worrying about gay object shit
-    const cardsCopy = JSON.parse(JSON.stringify(cards));
-
-    // render frames for the website
-    if (config.env != null &&
-      config.env.toLowerCase() === "dev" &&
-      message.content.split(" ")[1] === "dev-frames") {
-
-      renderFrames(cardsCopy);
-      return;
-    }
-
-    // legacy command support
-
-    // card <name>
-    // card <name> <level>
-    // card <name> l<level>
-    // card <name> u<upgrade>
-
-    let name = null;
-
-    let level = null;
-    let upgrade = null;
-
-    // 5 = card(4) + space(1).
-    const args = message.content.slice(guild.prefix.length + 5);
-    if (args === "") {
+  cb: async function (client, message) {
```

Comment:
define this function separately under the new Command() definition and pass it here instead, should be ok to make it an arrow function as well

Reply:
???

Reply:
i mean something like this
![image](https://user-images.githubusercontent.com/18635887/71322041-506c1900-24ba-11ea-8abc-0844d3200480.png)

Reply:
![image](https://user-images.githubusercontent.com/54243490/71323058-0a698200-24c7-11ea-8d7b-35585ea3d415.png)
uhh what do you mean with this, just name card cb?

Reply:
Just make the entirety of what you're setting the cb field to a fn defined in the global scope and specify that

const cb = your fn

card {
  cb: cb
}

export card